### ISSUE TITLE
[Bug]: TestHandler_Handle ci ut fail

### ISSUE THREAD

**Issue by XuPeng-SH** [matrixorigin/matrixone#9027]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

<img width="1240" alt="image" src="https://user-images.githubusercontent.com/39627130/232498629-9056ac8a-e3df-4599-b46c-a98a9759cfc7.png">

https://github.com/matrixorigin/matrixone/actions/runs/4721610913/jobs/8375090237?pr=9025

**Comment by volgariver6** [comment]
fixed by #9050.

**Comment by XuPeng-SH** [comment]
https://github.com/matrixorigin/matrixone/actions/runs/4777725185/jobs/8493658926?pr=9132

**Comment by XuPeng-SH** [comment]
https://github.com/matrixorigin/matrixone/actions/runs/4778439978/jobs/8494862332?pr=9132

**Comment by volgariver6** [comment]
fixed by #9137.

**Comment by volgariver6** [comment]
still not fixed, I will handle it.

**Comment by volgariver6** [comment]
fixed by #9161.
